index.html

```diff
@@ -4,63 +4,219 @@
+                    <p class="content">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <nav>
+                        <ul>
+                            <li>
+                                <a href="" target="_blank">
+                                    <img src="./assets/img-block1/html.png" alt="">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="" target="_blank">
+                                    <img src="./assets/img-block1/css.png" alt="">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="" target="_blank">
+                                    <img src="./assets/img-block1/javascript.png" alt="">
+                                </a>
+                            </li>
+                        </ul>
+                    </nav>
+                    <div>
+                        <button type="button" class="see_button">See Project</button>
+                    </div>
+                </div>                
+            </section>
+    <!--card_4 starts from here-->
+            <section class="picture_container">
+                    <img src="./assets/pictures/picture4.jpg" alt=""/>
+                <div class="info_container">   
+                    <h1 class="title">Multi-Post<br>Stories</h1>
+                    <div class="tonic_1">
+                        <img src="./assets/img-block1/Frame1.png" alt="">   
+                    </div>
+                    <p class="content">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <nav>
+                        <ul>
+                            <li>
+                                <a href="" target="_blank">
+                                    <img src="./assets/img-block1/html.png" alt="">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="" target="_blank">
+                                    <img src="./assets/img-block1/css.png" alt="">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="" target="_blank">
+                                    <img src="./assets/img-block1/javascript.png" alt="">
+                                </a>
+                            </li>
+                        </ul>
+                    </nav>
+                    <div>
+                        <button type="button" class="see_button">See Project</button>
+                    </div>
+                </div>                
+            </section>
+<!--about myself section from here-->
+            <div id="myself_div">    
+                <section class="picture_container">
```

Comment:
- [ ] You should use other than `<section>` here if you need to have a container. `<div>` can be used here.